### ISSUE TITLE
Accessibility issue on Telerik Grid

### ISSUE THREAD

**Issue by AishwaryaR07** [telerik/kendo-angular#4140]
Hi Team,
 
I have faced the following violation (The role attribute is not necessary for the tr, th and td elements) in IBM accessibility checker of your accessibility sample and it meets only 61%. Is the bug?

Screenshot:

![Image](https://github.com/telerik/kendo-angular/assets/61075846/ec8f7b00-87d4-466e-92b2-eb4ece71f13d)

I am also noticed in your sample that you've used two separate tables for the header and content. Is there a particular reason for this choice, and does it offer any advantages or limitations?
